build/Makefile

```diff
@@ -41,8 +41,21 @@ SCRIPTDIR ?= $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 DSTDIR ?= $(PDO_INSTALL_ROOT)
 SRCDIR ?= $(abspath $(SCRIPTDIR)/..)
 
-KEYDIR := $(DSTDIR)/opt/pdo/keys/
-ETCDIR := $(DSTDIR)/opt/pdo/etc/
+# this is an optional configuration variable to control the
+# hostname used in configuration files; we use it here to
+# customize the site.psh file
+ifndef PDO_HOSTNAME
+SERVICE_HOST := localhost
+else
+SERVICE_HOST := $(PDO_HOSTNAME)
+endif
+
+# splitting out the configuration root in order to make it easier
+# to build a separate configuration that can be used with the startup
+# scripts; by default the configuration is put into the regular tree
+CONFDIR := $(DSTDIR)
```

Comment:
Shouldn't this be surrounded by an `ifndef` statement that defaults to `DSTDIR`?

Reply:
This was supposed to be a conditional assignment. Will fix.

Reply:
Done